.travis/release_spark_enrich.yml

```diff
@@ -15,26 +15,26 @@ targets:
 # Required: packages to be deployed
 packages:
   # AWS S3 plain upload example
-  - name     : "hadoop-shred-hosted-asset"        # Just for convenience
+  - name     : "spark-enrich-hosted-asset"        # Just for convenience
     bucket   : "snowplow-hosted-assets"
-    path     : "3-enrich/scala-hadoop-shred"
+    path     : "3-enrich/spark-enrich"
     region   : "us-west-1"
     publish  : true
     override : false
     continue_on_conflict : false
-    version  : <%= FUNC['sbt_version(3-enrich/scala-hadoop-shred/)'] %>
+    version  : <%= FUNC['sbt_version(3-enrich/spark-enrich/)'] %>
 
     # Required IF '--check-version' is passed: will assert that both versions are the same
     build_version : <%= ENV['TRAVIS_BUILD_RELEASE_TAG'] %>
 
     # Optional: Build commands
     build_commands:
-      - cd 3-enrich/scala-hadoop-shred && sbt assembly && cd ../..
```

Comment:
I think we also need to have `cd 3-enrich/scala-common-enrich && sbt publishLocal`. Otherwise it'll fail to fetch right dependencies. Same for rdb shredder and all other applications/jobs.